pygeoapi/provider/postgresql.py

```diff
@@ -310,7 +317,7 @@ def _reflect_table_model(self, engine):
 
         return TableModel
 
-    def _sqlalchemy_to_feature(self, item):
+    def _sqlalchemy_to_feature(self, item, crs_transform_out):
```

Comment:
It could, but does not need to. If no transformation needs to be done, `crs_transform_out` is already assigned `None` (return value of the `_get_crs_transform` method if `crs_transform_wkt` is `None`). Same goes for the `crs_transform_wkt` parameter of the `_get_crs_transform` method. There, the default value is set to `None`, but it does need to, as its value is inherited from the same named parameter of either `get` or `query`.

Reply:
OK, then it's up to you to decide for the explicit/implicit way to go